go/engine/pgp_pull.go

```diff
@@ -128,20 +127,71 @@ func (e *PGPPullEngine) getTrackedUserSummaries(ctx *Context) ([]keybase1.UserSu
 	return matchedList, nil
 }
 
-func (e *PGPPullEngine) Run(ctx *Context) error {
-	summaries, err := e.getTrackedUserSummaries(ctx)
+func (e *PGPPullEngine) runLoggedOut(ctx *Context) error {
+	if len(e.userAsserts) == 0 {
```

Comment:
Does this not also get checked in the logged in version? What does it mean to pull a PGP key with no user?